webpack.config.js

```diff
@@ -31,6 +31,19 @@ module.exports = (env, options) => ({
     },
     module: {
         rules: [
+            // handle .woff and .woff2 fonts
+            {
+                test: /\.(woff(2)?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
+                use: [
+                    {
+                        loader: "file-loader",
+                        options: {
+                            name: "[name].[ext]",
+                            outputPath: "fonts/",
```

Comment:
does this mean fonts will still be included in the webpack bundle? what does that mean for restricting access by referrer?

Reply:
I think you're right. And actually, this part might not be needed—it's working fine without it for me!